src/tests/CMakeLists.txt

```diff
@@ -93,7 +93,17 @@ if(ENABLE_TESTING)
     target_compile_definitions(unit_${PROG} PRIVATE BOOST_TEST_DYN_LINK)
     file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Run_unit_${PROG})
     add_test(NAME unit_${PROG} COMMAND $<TARGET_FILE:unit_${PROG}> WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/Run_unit_${PROG})
-    # run tests for xtp (for coverage) as well
     set_tests_properties(unit_${PROG} PROPERTIES LABELS "xtp;votca;unit")
   endforeach(PROG)
-endif()
+
+  add_test(
```

Comment:
I would move this to the the pyxtp directory. This directory is more for tests of libxtp.

Reply:
I have done so, thanks for the suggestion!